### PR TITLE
Added a new module to fetch Avi Controller version and data

### PR DESCRIPTION
**Pull request by grastogi23**
##### SUMMARY

Added a new module to fetch Avi Controller version and data

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

avi_api_version
##### ANSIBLE VERSION

```
ansible 2.5.0
  config file = /home/grastogi/.ansible.cfg
  configured module search path = [u'/home/grastogi/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/grastogi/trace/ansible2.5/local/lib/python2.7/site-packages/ansible
  executable location = /home/grastogi/trace/ansible2.5/bin/ansible
  python version = 2.7.6 (default, Jun 22 2015, 17:58:13) [GCC 4.8.2]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by khaltore** [comment]
shipit

File: lib/ansible/modules/network/avi/avi_api_version.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+module: avi_api_version
+author: Vilian Atmadzhov (vilian.atmadzhov@paddypowerbetfair.com)
+
+short_description: Avi API Version Module
+description:
+    - This module can be used to obtain the version of the Avi REST API. U(https://avinetworks.com/)
+version_added: 2.5
+requirements: [ avisdk ]
+options:
+    tenant:
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_api_version.py]: This isn't defined in the argspec

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/avi/avi_api_version.py (hunk 1 above)
Comment (grastogi23) [reply lib/ansible/modules/network/avi/avi_api_version.py]: @gundalow wasn't sure if options is required. I have it taken out in next commit.

Comment (gundalow) [reply lib/ansible/modules/network/avi/avi_api_version.py]: If the module doesn’t doesn’t have any options (for example, it’s a _facts module), you can use `options: {}`.

From http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html

**Pull request merged**